pkg/propertiesformatter/summaryFormatter.go

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package propertiesformatter
+
+import (
+	"strconv"
+	"strings"
+)
+
+// -----------------------------------------------------
+// Summary format.
+const (
+	SUMMARY_FORMATTER_NAME = "summary"
+)
+
+type PropertySummaryFormatter struct {
+}
+
+func NewPropertySummaryFormatter() PropertyFormatter {
+	return new(PropertySummaryFormatter)
+}
+
+func (*PropertySummaryFormatter) GetName() string {
+	return SUMMARY_FORMATTER_NAME
+}
+
+func (*PropertySummaryFormatter) IsNeedingMethodDetails() bool {
+	return false
+}
```

Comment:
Don't think we need this for formatting properties since it looks like a runs-specific method